requirements.txt

```diff
@@ -9,6 +9,11 @@ dipy!=1.6.*,!=1.7.*,<1.9.0
 --extra-index-url https://download.pytorch.org/whl/cpu # append_to_freeze
 ivadomed
 matplotlib
+# `matplotlib-inline` is an optional package that comes with `jupyter` for use with the '%matplotlib inline' directive.
+# However, if the user runs `jupyter` from a separate environment and sets '%matplotlib inline', then SCT's
+# environment will be missing this backend, and SCT scripts will crash when trying to import matplotlib. So, we install
+# `matplotlib-inline` in our env by default to ensure that SCT scripts can be run in Jupyter notebooks without error.
+matplotlib-inline
```

Comment:
Ahhh... it looks like this is problematic from the perspective of `sct_check_dependencies`, which tries to import `matplotlib-inline`. 

There are two issues with this:

- `matplotlib-inline` maps to `matplotlib_inline` (easily fixable)
- `import matplotlib_inline` requires IPython. (However, implicitly using `matplotlib inline` via a Jupyter notebook does _not_ require IPython to be installed in SCT's env.)

So, `matplotlib-inline` should really be skipped when checking versions, since we probably don't want to install `IPython` just to import `matplotlib-inline`.